src/pages/en/guides/typescript.md

```diff
@@ -28,14 +28,29 @@ Some TypeScript configuration options require special attention in Astro. Below
     "resolveJsonModule": true,
     // Enable stricter transpilation for better output.
     "isolatedModules": true,
-    // Add type definitions for our Astro runtime.
-    "types": ["astro/client"],
-    // Tell TypeScript where your build output is
-    "outDir": "./dist"
+    // Astro will directly run your TypeScript code, no transpilation needed.
+    "noEmit": true
   }
 }
 ```
 
+Additionally, our templates include an `env.d.ts` file inside the `src` folder to provide [Vite's client types](https://vitejs.dev/guide/features.html#client-types) to your project:
```

Comment:
Is it "an `env.d.ts` file" or "a `env.d.ts` file"?

Reply:
I would pretty confidently say "an" for English if it's not written with a "dot" before the env. It's mostly when files start with a dot then a vowel that it can get tricky.  😅